auacm/app/static/js/controllers/SettingsController.js

```diff
@@ -33,4 +34,44 @@ app.controller('SettingsController', ['$scope', '$http', function($scope, $http)
             console.log("error");
         });
     }
+
+    $scope.createUser = function() {
+        console.log($scope.newUserDisplayName);
+        // Minimal error checking (should do a lot more)
+        if ($scope.newUserPassword.length < 6) {
+            return; // "should probably tell the user no"
+        }
+        if ($scope.newUserPassword !== $scope.newUserPasswordConfirm) {
+            return; // "tell the user that the passwords did not match"
+        }
+        var request = {
+            'username' : $scope.newUserName,
+            'password' : $scope.newUserPassword,
+            'display'  : $scope.newUserDisplayName
+        };
+
+        // Make the call to the api
+        $http({
+            method: 'POST',
+            url: '/api/create_user',
+            headers: {'Content-Type': 'application/x-www-form-urlencoded'},
+            transformRequest: function(obj) {
+                var str = [];
+                for (var p in obj)
+                    str.push(encodeURIComponent(p) + "=" + encodeURIComponent(obj[p]));
+                console.log("Request: " + str.join("&"));
+                return str.join("&");
+            },
+            data: request
+        }).then(function(response) {
+            if (response.status == 200) {
```

Comment:
Ok, I was following your example from lines 25-35. Should I change those too? Is there every an instance when the status code would be 201-299 and we would need to report something wrong?